livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LivestreamComposer.kt

```diff
@@ -35,43 +31,36 @@ fun LivestreamComposer(
             .fillMaxWidth(),
         verticalAlignment = Alignment.CenterVertically
     ) {
-        MessageInput(
+        MessageComposer(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(start = 8.dp)
-                .weight(1f),
-            value = composerViewModel.input,
-            attachments = emptyList(),
-            activeAction = composerViewModel.activeAction,
-            onValueChange = { composerViewModel.setMessageInput(it) },
-            onAttachmentRemoved = {},
-            label = {
-                Row(
-                    Modifier.wrapContentWidth(),
-                    verticalAlignment = Alignment.CenterVertically
-                ) {
-                    Text(
-                        modifier = Modifier.padding(start = 4.dp),
-                        text = "Add your comments here",
-                        color = ChatTheme.colors.textLowEmphasis
-                    )
-                }
-            }
-        )
-        val isInputValid = composerViewModel.input.isNotEmpty()
-        IconButton(
-            enabled = isInputValid,
-            content = {
-                Icon(
-                    imageVector = Icons.Default.Send,
-                    contentDescription = stringResource(id = R.string.stream_compose_send_message),
-                    tint = if (isInputValid) ChatTheme.colors.primaryAccent else ChatTheme.colors.textLowEmphasis
+                .wrapContentHeight(),
+            viewModel = composerViewModel,
+            integrations = {},
+            input = {
+                MessageInput(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .weight(1f)
+                        .padding(start = 8.dp),
+                    value = composerViewModel.input,
+                    attachments = composerViewModel.selectedAttachments,
+                    activeAction = composerViewModel.activeAction,
+                    onValueChange = { composerViewModel.setMessageInput(it) },
+                    onAttachmentRemoved = { composerViewModel.removeSelectedAttachment(it) },
+                    label = {
+                        Row(
```

Comment:
Again, a wrapper component for a single Child. We shouldn't be doing this.

Reply:
Updated

Reply:
Closing this comment , in lieu of updated code 

this comment is pointing to outdated code